src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+
+    Animal.born(this);
+  }
 }
 
+Animal.alive = [];
+
+Animal.born = (animal) => {
+  Animal.alive.push(animal);
+};
```

Comment:
You don't need this, you may push immediately 
```suggestion
   Animal.alive.push(this);
  }
}

Animal.alive = [];
```